kmsauth/__init__.py

```diff
@@ -473,7 +473,7 @@ def get_token(self):
             if PY2:
                 token_bytes = bytes(token)
             else:
-                token_bytes = bytes(token, 'utf8')
+                token_bytes = token  # Already bytes in py3
```

Comment:
I get test failures unless this line is 
`token_bytes = token.encode("utf-8")`
I think this is because `token` by default is not a bytes-like object suitable for `b64encode` in python3